### PR TITLE
make initenv was mentioned twice in DEVELOP.md

### PR DESCRIPTION
**Pull request by arahuja**
[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/589)

### CODE REVIEW

**Comment by danvk** [comment]
@jaclynperrone your turn to do a review :)

File: DEVELOP.md [hunk 1]
```diff
@@ -43,7 +42,8 @@ source venv/bin/activate
 You will need to generate an `ENV.sh` file using 'make initenv'. `ENV.sh` contains environment variables that need to be exported. You may need to tweak some settings for your installation.
 
 ```
-make initenv
+make initenv                       # Initialize environment file.
+$EDITOR ENV.sh                     # Fill in values. (see Step #3)
```
Comment (jaclynperrone) [inline DEVELOP.md]: Great catch! We can now remove this "(see Step #3)".

**Comment by danvk** [comment]
Would one of you two like to merge this? :)

**Comment by jaclynperrone** [comment]
Merged!

On Thu, May 7, 2015 at 5:18 PM, Dan Vanderkam <notifications@github.com>
wrote:

> Would one of you two like to merge this? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/cycledash/pull/589#issuecomment-100022006>.
>



-- 
Jaclyn Perrone
jaclynperrone.com

**Pull request merged**